spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -153,7 +161,8 @@ def main(argv: Sequence[str]):
     param_centerline = ParamCenterline(
         algo_fitting=arguments.centerline_algo,
         smooth=arguments.centerline_smooth,
-        minmax=True)
+        minmax=True,
+        soft=arguments.centerline_soft)
 
     # Output folder
     if arguments.o is not None:
```

Comment:
> `-cm red` is good for binarized centerlines, but isn't the best for soft centerlines:

Good catch! We also noticed this! We were manually changing the colormap in FSLeyes to `red-yellow`.

> The only problem here is that for some reason, the display_viewer_syntax function only provides a subset of the valid FSLeyes colors. 
>
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a4430850c4068ad2da4b5c6ca247a017fa6e1b03/spinalcordtoolbox/utils/shell.py#L115-L116

Indeed, `dict_fsleyes` does not contain all available colormaps. All available colormaps can be obtained by running `fsleyes -h`. It would be great if we could fetch the list of all available colormaps directly from fsleyes instead of specifying them manually.

Reply:
Ah! If you were using red-yellow, then we can just use that for now, rather than reworking the colormap logic?

```python
    cm_ctl = 'red-yellow' if arguments.centerline_soft else 'red'
    display_viewer_syntax([fname_input_data, file_output], colormaps=['gray', cm_ctl], opacities=['', '0.7'], verbose=verbose)
```

I'll open a new issue to address the limited colormap selection.

Reply:
> But, this isn't really a "true" segmentation... So, we might also want to look into the available options for ITKSnap to see if there are different flags we can use for soft segmentations.

To follow up on my comment about ITKSnap, it looks like there are no options for displaying a "soft" segmentation:

```
Image Options:
   -g FILE              : Load the main image from FILE
   -s FILE              : Load the segmentation image from FILE
   -l FILE              : Load label descriptions from FILE
   -o FILE [FILE+]      : Load additional images from FILE
                        :   (multiple files may be provided)
   -w FILE              : Load workspace from FILE
                        :   (-w cannot be mixed with -g,-s,-l,-o options)
```

So, I think sticking with `-s` is the best we can do on that front.

Reply:
> To follow up on my comment about ITKSnap, it looks like there are no options for displaying a "soft" segmentation:
> ....
> So, I think sticking with `-s` is the best we can do on that front.

I think so too.

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4026/commits/04681854bbeb35eaf1a0428d5567958a8bb98e61.